test/unsupported.spec.tsx

```diff
@@ -43,9 +46,19 @@ function setup(): Promise<ReactWrapper> {
 }
 describe('unsupported environments', () => {
   it("shouldn't break without DOM ", async () => {
-    const wrapper = await setup();
+    const mockOnLoad = jest.fn();
+
+    const wrapper = mount(
```

Comment:
Why didn't you use the setup here?

Reply:
As I removed `this.handleError` call from the implementation, the setup Promise would not resolve as neither `onError` or `onLoad` was called.